src/app/containers/StoryPromo/LinkContents/index.jsx

```diff
@@ -38,7 +38,7 @@ const LinkContents = ({ item }) => {
 
   // hilariously, this works. according to moment, null seconds == 0 seconds!
   const duration = moment.duration(rawDuration, 'seconds');
-  const durationString = formatDuration(duration, ',');
+  const durationString = formatDuration({ duration });
```

Comment:
I think I prefered `formatDuration` not taking in an object bcos now you'd have to change all it's usages which might lead to missing some passed arguments. the separator is not passed in here as it was before and the snapshots changed as well

Reply:
Updated